subsys/net/l2/ppp/Kconfig

```diff
@@ -11,6 +11,16 @@ menuconfig NET_L2_PPP
 
 if NET_L2_PPP
 
+config NET_L2_PPP_DELAY_STARTUP_MS
+	int "PPP delay startup ms"
+	default 0
+	help
+	  If the PPP starts too fast, it is possible to delay it
+	  a bit. This is mostly useful in debugging if you want the
+	  device be fully up before PPP handshake is started.
```

Comment:
I used the option for testing purposes. Normally the ppp comes up so fast that lot of logging is lost which was very annoying. So created this config option in order to help debugging things. Also enabled this option in echo-server sample because of that. Default value is 0 which means no delays.